test/encode_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule OpenApiSpex.EncodeTest do
+  use ExUnit.Case
+
+  alias OpenApiSpex.{
+    Info,
+    OpenApi,
+  }
+
+  test "Vendor extensions x-logo properly encoded" do
+    spec = %OpenApi{
+      info: %Info{
+        title: "Test",
+        version: "1.0.0",
+        extensions: %{
+          "x-logo" => %{
+            "url" => "https://example.com/logo.png",
+            "backgroundColor" => "#FFFFFF",
+            "altText" => "Example logo"
+    }
+        },
+      },
+      paths: %{ }
+    }
+
+    decoded = OpenApiSpex.resolve_schema_modules(spec)
+               |> Jason.encode!()
+               |> Jason.decode!()
+
+    assert "https://example.com/logo.png" == decoded["info"]["x-logo"]["url"]
```

Comment:
For consistency, can you reverse the arguments to `==` so that the expected value is on the right side?